source/reference/configuration.txt

```diff
@@ -12,16 +12,21 @@ Configuration
    :depth: 2
    :class: singlecol
 
-You can configure :program:`mongosync` instances at startup using a configuration
-file.  The configuration file contains settings that are the equivalent of
-``mongosync`` command-line options.
+You can configure :program:`mongosync` instances at startup using a
+configuration file.  The configuration file contains settings that are
+the equivalent of ``mongosync`` command-line options.
 
+The ``start`` command provides additional options for features like
```

Comment:
The update. 

I swapped the usual link, monospace order since the distance between the references  is short and 'See also' needs to have a link.